app/src/main/java/com/walmartlabs/classwork/rideone/util/Utils.java

```diff
@@ -0,0 +1,48 @@
+package com.walmartlabs.classwork.rideone.util;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.text.format.DateUtils;
+import android.view.Gravity;
+import android.widget.Toast;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+public class Utils {
+
+    //Check for internet connection
+    public static Boolean isNetworkAvailable(Context context) {
+        ConnectivityManager connectivityManager
+                = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo activeNetworkInfo = connectivityManager.getActiveNetworkInfo();
+        return activeNetworkInfo != null && activeNetworkInfo.isConnectedOrConnecting();
+    }
+
+    public static void showNoInternet(Context context) {
+        // Post a short message that app is operating in offline mode
+        Toast toast = Toast.makeText(context, "NO INTERNET CONNECTION", Toast.LENGTH_SHORT);
+        toast.setGravity(Gravity.CENTER_VERTICAL | Gravity.CENTER_HORIZONTAL, 0, 0);
+        toast.show();
+    }
+
+    // getRelativeTimeAgo("Mon Apr 01 21:16:23 +0000 2014");
+    public static String getRelativeTimeAgo(String rawJsonDate) {
+        String twitterFormat = "EEE MMM dd HH:mm:ss ZZZZZ yyyy";
+        SimpleDateFormat sf = new SimpleDateFormat(twitterFormat, Locale.ENGLISH);
```

Comment:
its not threadsafe so typically not a good idea to make it static. but in this case it can be made.
anyway its a copy-paste from one of the codepath material! :)